INSTALL.md

```diff
@@ -0,0 +1,34 @@
+# Installation from Sources
+
+Required Software:
```

Comment:
I think we should move build environment recommendations to another topic, probably where the library build is discussed. 
Also, we should decide if we want to provide the full list of supported (aka tested) systems or minimal requirements. 
@AlexVeprev @alexey-katranov

Reply:
Makes sense! I can delete it from there and update a PR with System Requirements topic.